src/components/PurchaseOrder/Summary/SummaryView.js

```diff
@@ -83,6 +84,18 @@ const SummaryView = ({ order, hiddenFields }) => (
           label={<FormattedMessage id="ui-orders.orderSummary.totalExpended" />}
         />
       </Col>
+
+      <Col
+        data-test-total-credited
+        xs={6}
+        lg={3}
+      >
+        <VersionKeyValue
```

Comment:
You must have wanted to put `VersionKeyValue` component [here](https://github.com/folio-org/ui-orders/blob/master/src/components/PurchaseOrder/POVersionView/SummaryVersionView/SummaryVersionView.js#L107-L108)?

Reply:
@usavkov-epam thanks, updated both places. Now should be good